### PR TITLE
Change function t

### PR DESCRIPTION
**Pull request by stone95**
Fixes #[7773](https://github.com/publiclab/plots2/pull/7773)  

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by nicoleiocana** [comment]
@stone95 Below are a few changes that the reviewers will request from you:

1. You might want to rename the commit message to something more appropriate (Change function t to translation in app/views/user_sessions/_form.html.erb) so that when merged the commit history of the repo looks good. The procedure for the same in git is using `git commit --amend`.
2. Could you please add the issue link up above on this line "Fixes #0000 (<=== Add issue number here)" to "Fixes 7737" , this will help other contributors while reviewing your changes. It also helps in automatically closing the issue when the pull request is merged.

I'm not an official reviewer, but I have seen these concerns pop up recently, so I figured that I should let you know. Congrats, you did a great job!

**Comment by stone95** [comment]
> @stone95 Below are a few changes that the reviewers will request from you:
> 
> 1. You might want to rename the commit message to something more appropriate (Change function t to translation in app/views/user_sessions/_form.html.erb) so that when merged the commit history of the repo looks good. The procedure for the same in git is using `git commit --amend`.
> 2. Could you please add the issue link up above on this line "Fixes #0000 (<=== Add issue number here)" to "Fixes 7737" , this will help other contributors while reviewing your changes. It also helps in automatically closing the issue when the pull request is merged.
> 
> I'm not an official reviewer, but I have seen these concerns pop up recently, so I figured that I should let you know. Congrats, you did a great job!

Thank you. This was very helpful to me! Best wishes to you and yours.

File: app/views/user_sessions/_form.html.erb [hunk 1]
```diff
@@ -3,10 +3,10 @@
 <input type="hidden" name="return_to" class="login-modal-redirect" value="<%= request.fullpath %>" />
 
 <center>
-  <h2><%= raw t('user_sessions.new.log_in') %></h2>
+  <h2><%= raw translate('user_sessions.new.log_in') %></h2>
```
Comment (cesswairimu) [inline app/views/user_sessions/_form.html.erb:6]: Hi @stone95 , thanks for opening this :tada: this should be changed to "translation". Thanks

Comment (cesswairimu) [reply app/views/user_sessions/_form.html.erb:6]: Also please change the link on `Fixed #... ` to the link to the issue.

**Comment by SidharthBansal** [comment]
As the person is inactive for more than a month, I am closing the PR. In case you want to push changes please feel free to open a new PR OR reopen this PR and add additional changes to it.
Thanks for contributing on Public Lab

**Pull request closed without merging**